src/pages/saksbehandling/steg/Vurdering.tsx

```diff
@@ -1,7 +1,11 @@
 import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
 import { Innholdstittel } from 'nav-frontend-typografi';
 import React from 'react';
+import { RawIntlProvider } from 'react-intl';
```

Comment:
Vad är skillnaden på `RawIntlProvider` och `IntlProvider` egentligen? ser att vi brukt bägge i projektet. 😅

Reply:
Hmm, litt usikker på. tror målet blir det samme, men litt annerledes setup mellom dem. De fungerer som wrappere for en undercomponent <FormatMessage /> (eller hva den het). Men ble litt mye kopiering etter å ha slått av hodet mitt i går, og tenkte ikke på at vi ikke trenger wrapperen.